drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -767,6 +771,19 @@ static void set_up_fixed_clock_sources(void)
 	}
 }
 
+/*
+ * MCO configure doesn't active requested clock source,
+ * so please make sure the clock source was enabled.
+ */
+static inline void stm32_clock_control_mco_init(void)
+{
+#if defined(CONFIG_CLOCK_STM32_MCO1_SRC_NOCLOCK)
```

Comment:
There should be a way not to add code when `CLOCK_STM32_MCO1_SRC_NOCLOCK` is selected.

Reply:
Hi, 

Can you please help me out? It has been a while and i forgot what has do be solved here. In case `CLOCK_STM32_MCO1_SRC_NOCLOCK` is selected, function `stm32_clock_control_mco_init()` shall be skipped at all?

```
/*
 * MCO configure doesn't active requested clock source,
 * so please make sure the clock source was enabled.
 */
static inline void stm32_clock_control_mco_init(void)
{
#if defined(CONFIG_CLOCK_STM32_MCO1_SRC_NOCLOCK)
    LL_RCC_ConfigMCO(MCO1_SOURCE, mco1_prescaler(1));
#else
    LL_RCC_ConfigMCO(MCO1_SOURCE, mco1_prescaler(CONFIG_CLOCK_STM32_MCO1_DIV));
#endif /* CONFIG_CLOCK_STM32_MCO2_SRC_NOCLOCK */
}
```

Reply:
Hi, 

Can you please help me out? It has been a while and i forgot what has do be solved here. In case `CLOCK_STM32_MCO1_SRC_NOCLOCK` is selected, function `stm32_clock_control_mco_init()` shall be skipped at all?

```
/*
 * MCO configure doesn't active requested clock source,
 * so please make sure the clock source was enabled.
 */
static inline void stm32_clock_control_mco_init(void)
{
#if defined(CONFIG_CLOCK_STM32_MCO1_SRC_NOCLOCK)
    LL_RCC_ConfigMCO(MCO1_SOURCE, mco1_prescaler(1));
#else
    LL_RCC_ConfigMCO(MCO1_SOURCE, mco1_prescaler(CONFIG_CLOCK_STM32_MCO1_DIV));
#endif /* CONFIG_CLOCK_STM32_MCO2_SRC_NOCLOCK */
}
```